### PR TITLE
Rtrieu/docs 4390 define apps services components rum

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do?
Adds a new RUM guide about how to define applications, services, and components for RUM-monitored web applications.

### Motivation
[DOCS-4390](https://datadoghq.atlassian.net/browse/DOCS-4390)

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+- link: "/real_user_monitoring/browser/"
+  tag: "Documentation"
+  text: "RUM Browser Monitoring"
+---
+
+## Overview
+
+RUM lets you capture all events in your browser applications and explore them to troubleshoot slow pages and code errors, or to analyze application usage. All events are available in [RUM Explorer][1] for querying, dashboarding, and alerting.
+
+If your browser application is large, it's likely been built thanks to multiple web development teams. Each team has an area of ownership that they focus on when troubleshooting errors, slowness, or analyzing usage.
```
Comment (cswatt) [inline content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md:15]:
```suggestion
If your browser application is large, it's likely been built by multiple web development teams. Each team has an area of ownership that they focus on when troubleshooting errors, slowness, or analyzing usage.
```

File: content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+This guide describes how to define an application in RUM. In addition, it covers common use cases in large applications where web development teams may require visibility into the health and usage of their area of ownership.
+
+## Create a RUM application
+
+The first step to tracking and analyzing your browser application is to [create a RUM application][2]. A RUM application maps a browser application available at a given domain that renders the experience for what customers would perceive as a website.
+
+For example, if your browser application is available at `https://www.yourwebsite.com`, create a RUM application called "Your Website".
+
+If your browser has subdomains, like `https://www.account.yourwebsite.com`, you should onboard one RUM application for your website.
```
Comment (cswatt) [inline content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md:25]: I'm confused by this sentence—browsers don't have subdomains. Also, what does it mean to "onboard" a RUM application? Can we just say "create"?

I'm wondering if what was meant by this sentence is "If your browser application has subdomains, like `https://www.account.yourwebsite.com`, create one RUM application for each subdomain."

File: content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+## Track pages in your browser application
+
+Whether your browser application is a single page application or is one that uses server-side rendering, the Browser RUM SDK automatically tracks route changes and creates a view event for every route change.
+
+- A view has a **URL** available at `@view.url`, such as `https://www.yourwebsite.com/about`.
+- A view has a **path** available at `@view.url_path`, such as `/about`.
+
+If, for example, automatically capturing page views by route change does not provide enough visibility, you can specify a different name for your pages. To do this, you can [track views manually][3] and assign them a name available at `@view.name`, such as "About Us".
```
Comment (cswatt) [inline content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md:34]:
```suggestion
If, for example, automatically capturing page views by route change does not provide enough visibility, you can specify a different name for your pages. To do this, you can [track views manually][3] and assign them each a name available at `@view.name`, such as "About Us".
```

File: content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+## Track timings during the rendering lifecycle of your pages
+
+The Browser SDK automatically tracks a set of latencies, or timings, that are industry standard. These include the Core Web Vitals, page loading times, [and more][4].
```
Comment (cswatt) [inline content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md:38]:
```suggestion
The Browser SDK automatically tracks a set of latencies, or timings, that are industry standards. These include the Core Web Vitals, page loading times, [and more][4].
```

File: content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+{{< img src="real_user_monitoring/guide/define-applications-services-components-rum/rum-guide-track-team-ownership-1.jpg" alt="Examples of sets of pages a web development could own" style="width:90%;">}}
+
+Inside your RUM application, create services for each set of pages owned by a team by doing the following:
+
+1. Turn on manual view tracking by setting the configuration option `trackViewsManually` to `true`.
+2. For each page of your website, assign a view name and a service following [these instructions][8].
+   - `"purchase"` service for the pages available at `/checkout`, `/payment`, `/confirmOrder`.
+   - `"catalog"` service for the pages available at `/beds`, `/chairs/123`, `/search`.
+3. [Upload a sourcemap for each service][9] to view unminified stack traces in Error Tracking.
```
Comment (cswatt) [inline content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md:101]:
```suggestion
3. [Upload a source map for each service][9] to view unminified stack traces in Error Tracking. 
```

File: content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+
+Inside your RUM application, create services for each set of pages owned by a team by doing the following:
+
+1. Turn on manual view tracking by setting the configuration option `trackViewsManually` to `true`.
+2. For each page of your website, assign a view name and a service following [these instructions][8].
+   - `"purchase"` service for the pages available at `/checkout`, `/payment`, `/confirmOrder`.
+   - `"catalog"` service for the pages available at `/beds`, `/chairs/123`, `/search`.
+3. [Upload a sourcemap for each service][9] to view unminified stack traces in Error Tracking. 
+
+Get insights into the performance or the adoption of a given team's scope by using the service attribute in RUM:
```
Comment (cswatt) [inline content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md:103]:
```suggestion
Get insights into the performance or the adoption of a given team's scope by using the `service` attribute in RUM:
```

**Review by cswatt — CHANGES_REQUESTED** [review]

File: content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+In addition, you can track the time it takes for a specific item on the page to render, such as an image or a component. You can track more timings by capturing them in code, then pasting the values in your view events, as [documented here][5].
+
+Once timings are captured, they are available like any auto-collected timing. You can use timings to do the following:
+
+- Analyze the distribution of the time across versions of the code in the RUM Explorer
+- Troubleshoot possible high values in the [view waterfall][6]
+
+## Track components in web pages
+
+If your browser application leverages UI components that are used across multiple pages in one application and/or across multiple applications, you can track the usage of components across pages, as well as the time they take to render, through custom instrumentation.
```
[the remark this answers is not in the document]
Comment (rtrieu) [reply content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md:49]: Oh good to know about this word, will note this!

File: content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md (hunk 2 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/guide/define-services-and-track-ui-components-in-your-browser-application.md]: Edited in file. Thanks for catching this!

**Pull request merged**